resources/Qtmodels/EditComponentWindow.qml

```diff
@@ -79,12 +87,12 @@ ExpandingWindow {
                 Label {
                     id: transformLabel
                     text: "Transform:"
-                    Layout.columnSpan: 2
+                    Layout.columnSpan: 3
```

Comment:
As you want it to span all columns perhaps it is better to use `parent.columns` rather than hardcoding as `3`?